packages/react-celo/src/connectors/connectors.ts

```diff
@@ -134,7 +134,7 @@ export class LedgerConnector implements Connector {
     localStorage.setItem(localStorageKeys.lastUsedNetwork, network.name);
     this.kit = newKit(network.rpcUrl);
   }
-
+  // maybe Connectors shouldnt be in charge of remembering persistent state
```

Comment:
I think this is debatable, each connector may need something different in order to re-create itself from localstorage when reloading the page

Reply:
ok, yeah that does make sense. I think what would be good is for there to just be one place that updates local storage.